### PR TITLE
anki, dig: add Turkish translation

### PR DESCRIPTION
**Pull request by ilkermeliksitki**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr, seems like you are creating new branches in your fork keeping your existing main branch as the base, in the future I would suggest using `tldr-pages:main` as the base branch. I have closed your other PR.

File: pages.tr/common/dig.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# dig
+
+> DNS sunucularına sorgulama yapmak için kullanılan bir komuttur.
+> Daha fazla bilgi için: <https://manned.org/dig>
```
Comment (kbdharun) [inline pages.tr/common/dig.md:4]:
```suggestion
> Daha fazla bilgi için: <https://manned.org/dig>.
```

File: pages.tr/common/anki.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# anki
+
+> Güçlü ve akıllı bir aralıklı tekrar programı.
+> Daha fazla bilgi için: <https://docs.ankiweb.net>.
+
+- Çalıştır.
```
Comment (kbdharun) [inline pages.tr/common/anki.md:6]:
```suggestion
- Çalıştır:
```

File: pages.tr/common/anki.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki`
+
+- Belirtilen bir profil ismi ile çalıştır.
```
Comment (kbdharun) [inline pages.tr/common/anki.md:10]:
```suggestion
- Belirtilen bir profil ismi ile çalıştır:
```

File: pages.tr/common/anki.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki -p {{profile_name}}`
+
+- Belirtilen bir dil ile çalıştır.
```
Comment (kbdharun) [inline pages.tr/common/anki.md:14]:
```suggestion
- Belirtilen bir dil ile çalıştır:
```

File: pages.tr/common/anki.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki -l {{language}}`
+
+- Belirtilen bir dizinden (konumdan) calıştır.
```
Comment (kbdharun) [inline pages.tr/common/anki.md:18]:
```suggestion
- Belirtilen bir dizinden (konumdan) calıştır:
```

File: pages.tr/common/dig.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- İlgili sunucu ile ilgili IP adresi sorgulaması yapılır.
```
Comment (kbdharun) [inline pages.tr/common/dig.md:6]:
```suggestion
- İlgili sunucu ile ilgili IP adresi sorgulaması yapılır:
```

File: pages.tr/common/dig.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`dig +short {{example.com}}`
+
+- DNS sorgulaması için alternatif server kullanılır.
```
Comment (kbdharun) [inline pages.tr/common/dig.md:10]:
```suggestion
- DNS sorgulaması için alternatif server kullanılır:
```

File: pages.tr/common/dig.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`dig @{{1.1.1.1}} {{example.com}}`
+
+- Tersine DNS sorgulaması yapılır.
```
Comment (kbdharun) [inline pages.tr/common/dig.md:14]:
```suggestion
- Tersine DNS sorgulaması yapılır:
```

File: pages.tr/common/dig.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`dig -x 1.1.1.1`
+
+- Belli başlı DNS sunucuları ile ilgili kayıtları sorgular.
```
Comment (kbdharun) [inline pages.tr/common/dig.md:18]:
```suggestion
- Belli başlı DNS sunucuları ile ilgili kayıtları sorgular:
```

File: pages.tr/common/dig.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+`dig +short {{example.com}}`
+
+- DNS sorgulaması için alternatif server kullanılır.
+
+`dig @{{1.1.1.1}} {{example.com}}`
+
+- Tersine DNS sorgulaması yapılır.
+
+`dig -x 1.1.1.1`
```
Comment (kbdharun) [inline pages.tr/common/dig.md:16]:
```suggestion
`dig -x {{1.1.1.1}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. **Also, wherever possible I would suggest translating the placeholder inside `<>` too.**

**Comment by kbdharun** [comment]
Tip: In the file tab, you can batch all suggestions and make a single commit.

**Review by Magrid0 — APPROVED** [review]

**Comment by ilkermeliksitki** [comment]
Thanks for your fixes. So, should I use my main branch that is based on tldr-pages:main while making a PR?  @kbdharun

**Comment by kbdharun** [comment]
> Thanks for your fixes. So, should I use my main branch that is based on tldr-pages:main while making a PR? @kbdharun

Yes, when creating a new branch in future, you need to use it with `tldr-pages:main` as the source branch.

**Review by kbdharun — APPROVED** [review]

File: pages.tr/common/anki.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+`anki -p {{profile_name}}`
+
+- Belirtilen bir dil ile çalıştır:
+
+`anki -l {{language}}`
+
+- Belirtilen bir dizinden (konumdan) calıştır:
+
+`anki -b {{path/to/directory}}`
```
Comment (pixelcmtd) [inline pages.tr/common/anki.md:20]:
```suggestion
`anki -b {{dizin/yolu}}`
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Could you please translate the placeholders as well? I've added a suggestion for one of them

**Comment by ilkermeliksitki** [comment]
> Could you please translate the placeholders as well? I've added a suggestion for one of them

@pixelcmtd thank you for the suggestion. I have applied the change.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**